GenderPayGap.WebUI/Controllers/Account/ManageAccountController.cs

```diff
@@ -24,11 +27,22 @@ public IActionResult ManageAccountGet()
             // Get the current user
             User currentUser = ControllerHelper.GetGpgUserFromAspNetUser(User, dataRepository);
 
+            // Remove the email change request if it is expired
+            var showNewEmail = currentUser.NewEmailAddressRequestDate.HasValue && 
+                               currentUser.NewEmailAddressRequestDate.Value.AddDays(Global.EmailVerificationExpiryDays) >= VirtualDateTime.Now;
+            if (!showNewEmail)
+            {
+                currentUser.NewEmailAddress = null;
+                currentUser.NewEmailAddressRequestDate = null;
+                dataRepository.SaveChanges();
+            }

```

Comment:
Thoughts on doing this vs never resetting them? It doesn't make any practical difference anyway...

Reply:
I'm inclined to leave the new email address and request date as they are, because I always find it unexpected when a get endpoint modifies data.